### PR TITLE
captions added

### PR DESCRIPTION
**Pull request by khushmehn**

### CODE REVIEW

File: captions.csv [hunk 1]
```diff
@@ -8,10 +8,13 @@ Rolling with the homies.,Happy,"friends, people, fun",Positive
 The most important thing is to enjoy your life – to be happy – it’s all that matters.,Happy,"travel, work, scenary",Positive
 50% Savage. 50% Sweetness.,Neutral,"Selfie, person, badass, cute",Neutral
 Watch more sunsets than Netflix.,Happy,"Calm, sunset, view, mountains",Positive
-"Love can be unselfish, in the sense of being benevolent and generous, without being selfless.",,,Positive
-"When I feel a little down, I put on my favourite high heels and dance",,,Positive
-Pretty please with a cherrry on top,,,Positive
```
Comment (devcer) [inline captions.csv:13]: Don't remove the existing captions.

**Comment by devcer** [comment]
@khushmehn 
* Update with master and Resolve conflicts
* Check the updated description on the issue and make sure your PR qualifies for all the tasks mentioned in the issue

File: captions.csv (hunk 1 above)
Comment (khushmehn) [reply captions.csv:13]: ok i will add more captions without removing others

**Comment by devcer** [comment]
Doesn't follow the guidelines set in https://github.com/devcer/captions-for-ig/issues/1 ! Please resubmit to get this merged.

**Pull request closed without merging**